### PR TITLE
Adding bootstrap during install/upgrade

### PR DESCRIPTION
**Pull request by kwoodson**
This pull request enables the master bootstrapping functionality in install, upgrade, and entry point.

This includes:
- adding bootstrap settings to master-config.yaml
- generating and storing the kubeconfig
- generating node configmaps
- enabling the autoapprover statefulset

### CODE REVIEW

**Comment by kwoodson** [comment]
@michaelgugino,

This provides the ability to migrate a cluster from the old style of provisioning to the new golden image path where old style and new style can exist.  This enables the master to perform the certificate signing process to allow nodes into the cluster.

> Also, if this is being included in an upgrade, should we be disabling other items?
I'm not sure what should be disabled.  Did you have an example of this?  What did you have in mind?

File: playbooks/openshift-master/private/tasks/enable_bootstrap.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  - name: Setup the node group config maps
+    import_role:
+      name: openshift_node_group
+    run_once: True
+
+  - name: Setup the node bootstrap auto approver
+    import_role:
+      name: openshift_bootstrap_autoapprover
+    when:
```
Comment (DanyC97) [inline playbooks/openshift-master/private/tasks/enable_bootstrap.yml]: @kwoodson is this a typo and as such should be removed ?

Comment (kwoodson) [reply playbooks/openshift-master/private/tasks/enable_bootstrap.yml]: Thanks, updated.

File: playbooks/openshift-master/private/tasks/enable_bootstrap.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+- when: openshift_master_bootstrap_enabled | default(false) | bool
```
Comment (michaelgugino) [inline playbooks/openshift-master/private/tasks/enable_bootstrap.yml]: This boolean appears to be somewhat redundant.  Probably should remove for consistency; alternatively, remove the other bools.

Comment (kwoodson) [reply playbooks/openshift-master/private/tasks/enable_bootstrap.yml]: I can do that.

**Pull request merged**